src/hotspot/share/runtime/os.cpp

```diff
@@ -1889,6 +1889,9 @@ static void hemi_split(T* arr, unsigned num) {
 
 // Given an address range [min, max), attempts to reserve memory within this area, with the given alignment.
 // If randomize is true, the location will be randomized.
+#if defined(__clang__) || defined(__GNUC__)
+__attribute__((no_sanitize("undefined")))
+#endif
```

Comment:
I don't think this attribute addition should be made. I agree with @stefank that we should be eliminating the
potential overflow, since the compiler (without ubsan) is within its rights to discard a pointer overflow check,
since pointer overflow is UB.

Reply:
From what I saw and got info from Thomas,  my impression was that the method contains already overflow handling.
But if you think the current overflow handling is not 'good enough' , maybe it needs improvement.

Reply:
Yeah, I am more confused now than I was initially :) If you provide us with the requested info, maybe we can shed light onto this one.

I know that I spend a lot of time with this code, making sure all corner cases are clean. Therefore I am curious what ubsan digged up.